src/hw_stats.py

```diff
@@ -84,41 +156,67 @@ def get_hw_stats (fid, node_dut, cycle):
             for k in range (cfg.num_matrix):
                 for mvmu_t in mvmu_type:
                     # Xbar accesses
-                    for m in range(cfg.phy2log_ratio):
+                    if cfg.MVMU_ver == "Analog":
+                        for m in range(cfg.phy2log_ratio):
+                            if (mvmu_t == 'd'):
+                                for key,value in hw_comp_access['xbar_op'].items():
+                                    hw_comp_access['xbar_op'][key] += node_dut.tile_list[i].ima_list[j].matrix_list[k][mvmu_t][m].num_access[key]
+                            elif (mvmu_t == 'b'):
+                                for key,value in hw_comp_access['xbar_mtvm'].items():
+                                    hw_comp_access['xbar_mtvm'][key] += node_dut.tile_list[i].ima_list[j].matrix_list[k][mvmu_t][m].num_access[key]
```

Comment:
If we aren't changing the semantic for 'b' and 'd' mvmus, can we restore this to the earlier version ?